employee-rostering-server/src/main/java/org/optaweb/employeerostering/server/common/jaxrs/OptaWebExceptionMapper.java

```diff
@@ -16,35 +16,53 @@
 
 package org.optaweb.employeerostering.server.common.jaxrs;
 
-import java.io.PrintWriter;
-import java.io.StringWriter;
-
-import javax.persistence.EntityNotFoundException;
+import javax.inject.Inject;
 import javax.ws.rs.core.MediaType;
 import javax.ws.rs.core.Response;
 import javax.ws.rs.core.Response.Status;
 import javax.ws.rs.ext.ExceptionMapper;
 import javax.ws.rs.ext.Provider;
 
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.optaweb.employeerostering.server.exception.ExceptionDataMapper;
+import org.optaweb.employeerostering.server.exception.ExceptionDataMapper.ExceptionData;
+import org.optaweb.employeerostering.shared.exception.ServerSideExceptionInfo;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
 @Provider
 public class OptaWebExceptionMapper implements ExceptionMapper<Exception> {
```

Comment:
This class has no test coverage - please take a look at the SonarCloud coverate report. If you have any troubles seeing the report, let me know, I will be happy to help.

Reply:
It is partially covered in the integration test (although we don't inspect the JSON data in the integration test). I can add some JSON inspection tests to the integration tests (as well as create a unit test for it).